package.json

```diff
@@ -38,36 +40,36 @@
     "dist/transformer.cjs"
   ],
   "scripts": {
-    "build": "rollup -c rollup.config.js",
+    "build": "rollup -c rollup.config.cjs && cp src/preprocess.js dist/",
     "toc": "doctoc README.md",
     "lint": "standard --fix --env jest",
-    "test": "cross-env NODE_OPTIONS=--experimental-vm-modules jest src",
-    "test:watch": "npm run test -- --watch",
-    "test:update": "npm run test -- --updateSnapshot --coverage",
-    "setup": "npm i && npm run validate",
-    "validate": "npm run lint && npm run test",
+    "test": "npm run build && cross-env NODE_OPTIONS=--experimental-vm-modules jest src",
+    "test:nocache": "npm install && jest --clearCache && npm test -- --noCache",
+    "test:watch": "npm test -- --watch",
+    "test:update": "npm test -- --updateSnapshot --coverage",
+    "setup": "npm install && npm run validate",
+    "validate": "npm run lint && npm test:nocache",
     "release": "npm run validate && standard-version"
   },
   "peerDependencies": {
     "jest": ">= 27",
-    "svelte": ">= 3"
+    "svelte": ">= 3",
+    "svelte-preprocess": "<=4.7.4"
```

Comment:
I think this will force a dependency on `svelte-preprocess` even for projects written purely in JS that don't need to do any preprocessing

Reply:
4.8.0 works for me. I think the issue was just introduced last week: https://github.com/sveltejs/svelte-preprocess/blob/5aa60cb86c89c0649a548507f6d142937f76df7a/src/transformers/typescript.ts#L26

Reply:
I sent a PR to svelte-core that allowed me to track down this issue: https://github.com/sveltejs/svelte/pull/6722

Reply:
> I think this will force a dependency on `svelte-preprocess` even for projects written purely in JS that don't need to do any preprocessing

Ups, you are right.

Reply:
Fix for the `svelte-preprocess` issue is here: https://github.com/sveltejs/svelte-preprocess/pull/407. We can unpin `svelte-preprocess` in this project after that's committed